h2o-r/h2o-package/R/automl.R

```diff
@@ -486,3 +498,32 @@ h2o.getAutoML <- function(project_name) {
              training_info = training_info
   ))
 }
+
+#' Retrieve the leaderboard from the AutoML instance.
+#'
+#' Contrary to the default leaderboard attached to the automl instance, this one can return columns other than the metrics.
+#'
+#' @param aml A H2OAutoML instance for which to return the leaderboard.
+#' @param extensions A string or a list of string specifying which optional columns should be added to the leaderboard. Defaults to None.
+#' Currently supported extensions are:
+#' \itemize{
+#' \item{'ALL': adds all columns below.}
+#' \item{'training_time_ms': column providing the training time of each model in milliseconds (doesn't include the training of cross validation models).}
+#' \item{'predict_time_per_row_ms': column providing the average prediction time by the model for a single row.}
+#' }
+#' @return An H2OFrame representing the leaderboard.
+#' @examples
+#' \dontrun{
+#' library(h2o)
+#' h2o.init()
+#' votes_path <- system.file("extdata", "housevotes.csv", package = "h2o")
+#' votes_hf <- h2o.uploadFile(path = votes_path, header = TRUE)
+#' aml <- h2o.automl(y = "Class", project_name="aml_housevotes",
+#'                   training_frame = votes_hf, max_runtime_secs = 30)
+#' lb_all <- h2o.getLeaderboard(aml, 'ALL')
+#' lb_custom <- h2o.getLeaderboard(aml, c('predict_time_per_row_ms', 'training_time_ms'))
+#' }
+#' @export
+h2o.getLeaderboard <- function(aml, extensions=NULL) {
```

Comment:
@seb-h2o shouldn’t  we use underscore style `get_leaderboard` here?

Reply:
I've been hesitating too: problem is that automl module started to use `getAutoML`, so for consistency, using the same case here.
Another solution it to create an alias `get_automl` for `getAutoML` and have only `get_leaderboard` for this new function.
@ledell ?

Reply:
problem solved: 
- exposing `h2o.get_automl` and `h2o.get_leaderboard`
- deprecated `h2o.getAutoML`: still functional though.